service/controller/clusterapi/v29/controllercontext/cluster_az.go

```diff
@@ -0,0 +1,9 @@
+package controllercontext
+
+import "net"
+
+type ContextTenantClusterAvailabilityZone struct {
```

Comment:
I created this as a shared struct between `cc` `status` and `spec`. This made me slightly more sane and allows for some neat refactorings in follow up PRs.